packages/cli/src/commands/lint.ts

```diff
@@ -127,6 +135,11 @@ const lintCommand: CommandModule = {
           description: 'path/URL to a ruleset file',
           type: 'string',
         },
+        'scoring-config': {
+          alias: 's',
```

Comment:
tbh this option is rather quite specific, therefore I'm not sure whether an alias for it makes that much sense.
Would just keep it without an alias for now.